### ISSUE TITLE
Allow disabling swap detection

### ISSUE THREAD

**Issue by internalfx** [rethinkdb/rethinkdb#6018]
I understand why RethinkDB warns if the server is using swap. And I have taken steps to prevent swap usage on my production server.

However, on my development laptop this error is rather annoying and unnecessary since performance is not an issue.

I think a command line flag or some way to dismiss the error would be nice. I would prefer that my attention is directed to the big red bar only when there is actually a problem.

**Comment by deontologician** [comment]
Maybe we should have a UI option to suppress certain classes of issue. that
would be pretty easy

**Comment by danielmewes** [comment]
Yeah we should do something about this. An option in the UI might be enough.
